src/CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+
+### Code changes
+
+To make code changes, you will first need to locate the chapter you wish to edit in `/src/chapters`. Depending on the type of code change you'd like to make, you will need to use different commands.
+
+#### Running commands
+
+If you'd like to run a command, you will need to use the `run: command`. Be sure to include `cwd=super-rentals` so that the file is generated in the app's directory.
+
+For example:
+
+    ```run:command hidden=true cwd=super-rentals
+    ember generate component rental
+    ```
+
+#### Creating files
+
+To create a file, you will need to use the `run:file:create` command. Be sure to specify the language, the filename, as well as the `cwd` directory.
+
+For example:
+
+    ```run:file:create lang=handlebars cwd=super-rentals filename=app/templates/about.hbs
+    <h2>About Super Rentals</h2>
+    ```
+
+After creating a file, you should be sure to _add_ it shortly afterwards so that it is tracked by git, and will be committed in the process of generating the tutorial app. This can be a `hidden` command, since it is not part of the prose of the tutorial.
+
+For example:
+
+    ```run:command hidden=true cwd=super-rentals
+    git add app/templates/about.hbs
+    ```
+
+If you are adding a _test_ file, be sure to run `yarn test` before adding the test file. This way, the generator will run the tests before adding them, and will subsequently fail in generating broken code if the tests themselves fail.
+
+#### Using `run:pause`
+
+When running commands, generating new files, or making changes to preexisting ones, you may want to see the current state of the generated app. The `run: pause` command is useful in this case. Use this command inside of your existing chapter, inserting it at points where you want the generator to stop. This is similar to running something like `pauseTest()`; it will pause the generator and allow you to see the current state of the code you have generated (located in `/dist/code/super-rentals`).
+
+For example:
+
+    ```run:pause
+    Note to self: check that `application.hbs` has the correct markup and remember to git add it!
+    ```
+
+Any content inside of the code blocks will be printed in your command line.
+
+#### Generating diffs
+
+In the case of generating diffs to render inside of the tutorial prose, you will want to use the `run: pause` command, described above.
+
+_⚠️ Please note: you may find it easier to generate diffs and make edits to the tutorial by opening `/dist/code/super-rentals/code` in a separate code editor window._
+
+Once you have put a `run: pause` at the spot in your prose where the diff needs to be captured, you can navigate to `/dist/code/super-rentals/code` in the terminal. First, check that the diff is included in the current changes:
```

Comment:
I think you forgot to mention making the actual changes? (Also probably confirm git diff is empty before you start)